besu/src/main/java/org/hyperledger/besu/cli/options/unstable/SynchronizerOptions.java

```diff
@@ -58,6 +58,7 @@
       "--Xsynchronizer-world-state-min-millis-before-stalling";
   private static final String WORLD_STATE_TASK_CACHE_SIZE_FLAG =
       "--Xsynchronizer-world-state-task-cache-size";
+  private final String SNAP_ENABLED_FLAG = "--Xsnap-enabled";
```

Comment:
would it make sense for snap sync to be another option in --sync-mode ? 
FAST|FULL|SNAP